gotocompany/compass/v1beta1/service.proto

```diff
@@ -1146,6 +1161,24 @@ message DeleteAssetRequest {
 
 message DeleteAssetResponse {}
 
+message DeleteAssetsRequest {
+  string query_expr = 1 [
+    (validate.rules).string.min_len = 3,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "filter by query expr, example: refreshed_at <= \"2024-08-05T23:59:59\""
+    }
+  ];
+  bool dry_run = 2 [
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "if set to true, then deletion should not proceed and only return the affected rows. else, will perform deletion in the background"
+    }
+  ];
+}
+
+message DeleteAssetsResponse {
+  uint32 affected_rows = 1;
```

Comment:
should we add a field if the operation is `dry_run` or not? for the visibility to the user
and we will make the `dry_run` true by default right?

Reply:
By discussion with mas Haveiss, `dry_run` is false by default, so it really performs deletion if not specified.
And in response, we don't think it necessary because the client should know about it since he set it as true or false.